features/date.yml.dist

```diff
@@ -0,0 +1,185 @@
+  - javascript.builtins.Date.valueOf
+
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "1"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "1"
+  #   firefox_android: "4"
+  #   safari: "4"
+  #   safari_ios: "4"
+  - javascript.builtins.Date.now
+
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "3"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "1"
+  #   firefox_android: "4"
+  #   safari: "4"
+  #   safari_ios: "3.2"
+  - javascript.builtins.Date.toISOString
+  - javascript.builtins.Date.toJSON
+
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "6"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "4"
+  #   firefox_android: "4"
+  #   safari: "5.1"
+  #   safari_ios: "5"
+  - javascript.builtins.Date.parse.iso_8601
+
+  # baseline: high
+  # baseline_low_date: 2017-04-05
+  # baseline_high_date: 2019-10-05
+  # support:
+  #   chrome: "47"
+  #   chrome_android: "47"
+  #   edge: "15"
+  #   firefox: "44"
+  #   firefox_android: "44"
+  #   safari: "10"
+  #   safari_ios: "10"
+  - javascript.builtins.Date.@@toPrimitive
+
+  # baseline: high
+  # baseline_low_date: 2017-09-28
+  # baseline_high_date: 2020-03-28
+  # support:
+  #   chrome: "24"
+  #   chrome_android: "25"
+  #   edge: "12"
+  #   firefox: "29"
+  #   firefox_android: "56"
+  #   safari: "10"
+  #   safari_ios: "10"
+  - javascript.builtins.Date.toLocaleDateString.locales_parameter
+  - javascript.builtins.Date.toLocaleDateString.options_parameter
+  - javascript.builtins.Date.toLocaleString.locales_parameter
+  - javascript.builtins.Date.toLocaleString.options_parameter
+  - javascript.builtins.Date.toLocaleTimeString.locales_parameter
+  - javascript.builtins.Date.toLocaleTimeString.options_parameter
+
+  # baseline: high
+  # baseline_low_date: 2017-09-28
+  # baseline_high_date: 2020-03-28
+  # support:
+  #   chrome: "24"
+  #   chrome_android: "25"
+  #   edge: "14"
+  #   firefox: "52"
+  #   firefox_android: "56"
+  #   safari: "7"
+  #   safari_ios: "7"
+  - javascript.builtins.Date.toLocaleDateString.iana_time_zone_names
+  - javascript.builtins.Date.toLocaleString.iana_time_zone_names
+  - javascript.builtins.Date.toLocaleTimeString.iana_time_zone_names
+
+  # baseline: high
+  # baseline_low_date: 2020-01-15
+  # baseline_high_date: 2022-07-15
+  # support:
+  #   chrome: "57"
+  #   chrome_android: "57"
+  #   edge: "79"
+  #   firefox: "54"
+  #   firefox_android: "54"
+  #   safari: "12"
+  #   safari_ios: "12"
+  - javascript.builtins.Date.UTC.optional_monthIndex
+
+  # baseline: false
+  # support:
+  #   chrome: "1"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "1"
+  #   firefox_android: "4"
+  #   safari: "1"
+  #   safari_ios: "1"
+  - javascript.builtins.Date.getYear
```

Comment:
These are deprecated and `npm run dist` will omit a warning. Can you omit them so that we don't get a "Not Baseline" banner on these MDN pages? It's technically true that it's not Baseline, but it really should be a different category which doesn't exist yet.